OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -66,7 +66,7 @@ static string[] QueryDevices(string label, int type)
 			do
 			{
 				var str = Marshal.PtrToStringAuto(next);
-				next += UnicodeEncoding.Default.GetByteCount(str) + 1;
+				next += Encoding.Default.GetByteCount(str) + 1;
```

Comment:
Ok, so according to MSDN, `UnicodeEncoding` derives from `Encoding` and doesn't seem to overwrite the `Default` property, so `Encoding.Default` and `UnicodeEncoding.Default` are bound to be the same due to inheritance. This change therefore doesn't make any functional difference, it's still the same code in the end.